psi4/driver/diatomic.py

```diff
@@ -104,9 +109,17 @@ def anharmonicity(rvals, energies, plot_fit='', mol = None):
     m1 = molecule.mass(0)
     m2 = molecule.mass(1)
 
+    # Find rval of the minimum of energies, check number of points left and right
+    minE = np.min(energies)
```

Comment:
Can this be `minimum_energy`? Too many short hands as it is, can get to compressed. Ditto for `iminE`